src/app/layout/qgslayoutscalebarwidget.cpp

```diff
@@ -73,6 +83,14 @@ QgsLayoutScaleBarWidget::QgsLayoutScaleBarWidget( QgsLayoutItemScaleBar *scaleBa
   mStyleComboBox->insertItem( 4, tr( "Line Ticks Up" ) );
   mStyleComboBox->insertItem( 5, tr( "Numeric" ) );
 
+  //label vertical placement combo box
+  mLabelVerticalPlacementComboBox->insertItem( 0, tr( "Above segments" ) );
```

Comment:
Can you set the user data for these items as the corresponding enum values, and retrieve those (as is done elsewhere)?